interfaces/apparmor/backend.go

```diff
@@ -205,3 +205,7 @@ func unloadProfiles(profiles []string) error {
 	}
 	return nil
 }
+
+func (b *Backend) NewSpecification() interfaces.Specification {
+	panic(fmt.Errorf("%s is not using specifications yet", b.Name()))
```

Comment:
I was about to fix it but I'll propose a change that actually implements this correctly next so I won't bother.